account.js

```diff
@@ -0,0 +1,114 @@
+var readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+class BankAccount {
+  constructor(name, accountNumber, phoneNumber, amount, accountStatus) {
+    this.amount = amount;
+    this.name = name;
+    this.accountNumber = accountNumber;
+    this.phoneNumber = phoneNumber;
+    this.accountStatus = accountStatus;
+  }
+
+  open() {
+    var account = new BankAccount()
+    rl.question('Enter your name  :', function (name) {
+      rl.question('Enter accountNumber: ', function (accountNumber) {
+        rl.question('Enter phone number: ', function (phoneNumber) {
+          account.name = name
+          account.accountNumber = accountNumber
+          account.phoneNumber = phoneNumber
+          console.log(account)
+          console.log('account opened')
+        });
+      });
+    });
+
+  }
+  deposit(depositAmount, accountNumber) {
+    //rl.question('Enter account number  : ', function (accountNumber) {
+    if (this.accountNumber === accountNumber) {
+      // rl.question('Amount to deposit: ', function (depositAmount) {
+      this.amount = parseFloat(depositAmount);
```

Comment:
I have a question, If there is a balance of  say `5000` and someone deposits `10000` what will be the new balance given what you implemented here?